_pages/terse_guide.md

```diff
@@ -11,7 +11,7 @@ permalink:  /documentation/terse_guide/
 
 ## Variables
 - Variables must be declared before use
-- Shared vars must begin with uppercase
+- Shared vars (globa,s pools, and class vars) must begin with uppercase
```

Comment:
There seems to be a typo here.

Reply:
@LinqLover Is that a typo or some magic formatting I am not getting? If a typo, I can fix it and merge the pr.

Reply:
Oops, sorry :D

```suggestion
- Shared vars (globals, pools, and class vars) must begin with uppercase
```